donate/index.html

```diff
@@ -87,7 +87,7 @@ <h3 class="with-subtitle">What are you waiting for?</h3>
         <div class="container container-double">
             <h3 class="text-center red">Thank you!</h3>
             <p class="text-center">Thank you for your donation!</p>
-            <p class="text-center"><strong>Remember:</strong> You can get NewPipe for <strong>free</strong>.</p>
+            <p class="text-center"><strong>Remember:</strong> You can get NewPipe for <strong>free of cost</strong>.</p>
```

Comment:
I like that you want to clarify this line, but I do not like the wording.

What about 
```suggestion
            <p class="text-center"><strong>Remember:</strong> You can get NewPipe <strong>free of cost</strong>.</p>
```

I have never heard _for free of cost_ before. @opusforlife2 what's your opinion?

Reply:
For met "for free of costs" sounds not correct. But Google just states "you use for free".

However, maybe an entire different sentence: _NewPipe is provided without the need of any payment or purchase._

Reply:
To me, "without the need of any payment or purchase" sounds like we're discouraging donations.

Since the sentence is followed by "That's why any kind of help is welcome.", I would prefer:

"Remember: NewPipe is provided/made available to you gratis, or completely free of cost." to evoke the idea that there is labour going into this despite the fact that it is free.

Reply:
Yes, I saw that coming when I rewrote - but is of course not my intention. I am fine with your suggestion.

Reply:
If you agree I can change the PR

Reply:
sure give it a shot